infrastructure/base/linkerd.yaml

```diff
@@ -0,0 +1,1677 @@
+  install: |
+    {"uuid":"5ddcc6bb-795b-47fc-97dc-29ff4f6ce671","cliVersion":"stable-2.3.1","flags":[]}
+---
+###
+### Identity Controller Service
+###
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: linkerd-identity
+  namespace: linkerd
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: linkerd-linkerd-identity
+rules:
+- apiGroups: ["authentication.k8s.io"]
+  resources: ["tokenreviews"]
+  verbs: ["create"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: linkerd-linkerd-identity
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: linkerd-linkerd-identity
+subjects:
+- kind: ServiceAccount
+  name: linkerd-identity
+  namespace: linkerd
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: linkerd-identity
+  namespace: linkerd
+  labels:
+    linkerd.io/control-plane-component: identity
+  annotations:
+    linkerd.io/created-by: linkerd/cli stable-2.3.1
+spec:
+  type: ClusterIP
+  selector:
+    linkerd.io/control-plane-component: identity
+  ports:
+  - name: grpc
+    port: 8080
+    targetPort: 8080
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: linkerd-identity-issuer
+  namespace: linkerd
+  labels:
+    linkerd.io/control-plane-component: identity
+  annotations:
+    linkerd.io/created-by: linkerd/cli stable-2.3.1
+    linkerd.io/identity-issuer-expiry: 2020-06-02T04:19:02Z
+data:
+  crt.pem: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJoRENDQVNtZ0F3SUJBZ0lCQVRBS0JnZ3Foa2pPUFFRREFqQXBNU2N3SlFZRFZRUURFeDVwWkdWdWRHbDAKZVM1c2FXNXJaWEprTG1Oc2RYTjBaWEl1Ykc5allXd3dIaGNOTVRrd05qQXpNRFF4T0RReVdoY05NakF3TmpBeQpNRFF4T1RBeVdqQXBNU2N3SlFZRFZRUURFeDVwWkdWdWRHbDBlUzVzYVc1clpYSmtMbU5zZFhOMFpYSXViRzlqCllXd3dXVEFUQmdjcWhrak9QUUlCQmdncWhrak9QUU1CQndOQ0FBU2hQVG9lS1BuWVo0MVFPc3RVOTVRMnV2aVIKVVBhNVBMc3hZS0F3dFZZb0ZZbGNBdTA1aWZkY2VMNE5YWDUvTHEydWdDZmY0UVcyN2ZQMmtNZW4wKzVabzBJdwpRREFPQmdOVkhROEJBZjhFQkFNQ0FRWXdIUVlEVlIwbEJCWXdGQVlJS3dZQkJRVUhBd0VHQ0NzR0FRVUZCd01DCk1BOEdBMVVkRXdFQi93UUZNQU1CQWY4d0NnWUlLb1pJemowRUF3SURTUUF3UmdJaEFQM0JVM20yU1dYUkcxd3EKUjhkZ2ZHNkdpTFNrTHMxRU1CczdIZzVhRWVxZ0FpRUFwMmJwWFJMYkw3OTdvVjNiSk5mQjZYQkJ2di9JdFgrVgpBRjczZzZ6OHVUST0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
+  key.pem: LS0tLS1CRUdJTiBFQyBQUklWQVRFIEtFWS0tLS0tCk1IY0NBUUVFSUI2bVY5Ni80cFFtRkx2UHdXbG1EVFUyZDVicTBPMmZvRHA5L2t6NGg3Mk5vQW9HQ0NxR1NNNDkKQXdFSG9VUURRZ0FFb1QwNkhpajUyR2VOVURyTFZQZVVOcnI0a1ZEMnVUeTdNV0NnTUxWV0tCV0pYQUx0T1luMwpYSGkrRFYxK2Z5NnRyb0FuMytFRnR1M3o5cERIcDlQdVdRPT0KLS0tLS1FTkQgRUMgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
It's not ideal to have this as part of a public repo

Reply:
Wow. Well I guess there's a good argument for not committing code after midnight.
Thanks for pointing that out. I'll rework this and do some more digging into Linkerd!

Reply:
No worries, depends on just how much you value security 😃 

Feel free to ping us if you get stuck on anything. Big community on slack.linkerd.io